roles/sft-server/files/nginx.conf

```diff
@@ -0,0 +1,34 @@
+
+user  nginx;
+worker_processes  1;
+
+error_log  /var/log/nginx/error.log warn;
```

Comment:
Why not piping this to systemd, as you had suggested I do here: https://github.com/wireapp/ansible-restund/pull/11 ?

Reply:
Fixed.